runtime/redirector/forwarders.m4

```diff
@@ -413,5 +413,7 @@ _IF([JAVA_SPEC_VERSION >= 21],
 	[_X(JVM_VirtualThreadMount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide, jboolean firstMount)])
 _IF([JAVA_SPEC_VERSION >= 21],
 	[_X(JVM_VirtualThreadUnmount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide, jboolean lastUnmount)])
+_IF([JAVA_SPEC_VERSION >= 21],
+	[_X(JVM_IsForeignLinkerSupported, JNICALL, false, jboolean, void)])
```

Comment:
Please insert this addition before `JVM_VirtualThreadMount` (to keep the names for Java 21 sorted).

Reply:
Fixed.